anonlink/bloommatcher.py

```diff
@@ -1,8 +1,22 @@
+import inspect
+import warnings
+
 from anonlink._entitymatcher import ffi, lib
 
 __author__ = 'Stephen Hardy, Brian Thorne'
 
 
+def _fname():
+    return inspect.currentframe().f_back.f_code.co_name
+def _deprecation(use_instead=None):
+    msg = (f'anonlink.bloommatcher.{_fname()} has been deprecated ')
+    if use_instead is not None:
+        msg += '(use anonlink.{use_instead} instead)'
+    else:
+        msg += 'without replacement'
+    warnings.warn(msg, DeprecationWarning, stacklevel=2)
```

Comment:
Good work pointing at the replacement when available. 

Does the `stacklevel=2` mean the user will see their line of code that called your `_deprecation` function?